### ISSUE TITLE
False positive with "Dropping reference" inspection

### ISSUE THREAD

**Issue by oliver-giersch** [intellij-rust/intellij-rust#3606]
## Environment

* **Intellij-Rust plugin version:** v0.2.95.2120-183
* **Rust toolchain version:** 1.35.0-nightly
* **IDE name and version:** CLion 2018.3.4
* **Operating system:** Ubuntu 18.10

## Problem description

The plugin appears to confuse `std::mem::ManuallyDrop::drop` with `std::mem::drop` and reports a false positive when using the former.

## Steps to reproduce

Minimal example:

```rust
let mut drop = ManuallyDrop::new("nodrop");
unsafe { ManuallyDrop::drop(&mut drop) };
```
